log-elasticsearch/log-elasticsearch.cabal

```diff
@@ -24,16 +24,21 @@ Source-repository head
   Location: https://github.com/scrive/log.git
 
 library
-  exposed-modules:     Log.Backend.ElasticSearch
-                       Log.Backend.ElasticSearch.Internal
+  exposed-modules:     Log.Backend.ElasticSearch.V1
+                       Log.Backend.ElasticSearch.V1.Internal
+                       Log.Backend.ElasticSearch.V1.Lens
+                       Log.Backend.ElasticSearch.V5
+                       Log.Backend.ElasticSearch.V5.Internal
+                       Log.Backend.ElasticSearch.V5.Lens
+                       Log.Backend.ElasticSearch
```

Comment:
If the `L.B.ES.Internal` module is now gone, we'll have to do a major version bump. Probably safer to do one anyway.